examples/enclosure.go

```diff
@@ -20,7 +20,7 @@ func main() {
 		os.Getenv("ONEVIEW_OV_DOMAIN"),
 		os.Getenv("ONEVIEW_OV_ENDPOINT"),
 		false,
-		1000,
+		1200,
```

Comment:
same for enclosure, we have some new fields introduced for 1200 version

Reply:
No, we are just giving minimal support for 1200 with no changes to structs.